android/java/org/chromium/chrome/browser/brave_news/BraveNewsUtils.java

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.brave_news;
+
+import android.content.Context;
+
+import org.chromium.base.Log;
+import org.chromium.brave_news.mojom.Article;
+import org.chromium.brave_news.mojom.CardType;
+import org.chromium.brave_news.mojom.Deal;
+import org.chromium.brave_news.mojom.DisplayAd;
+import org.chromium.brave_news.mojom.FeedItem;
+import org.chromium.brave_news.mojom.FeedItemMetadata;
+import org.chromium.brave_news.mojom.PromotedArticle;
+import org.chromium.brave_news.mojom.Publisher;
+import org.chromium.chrome.browser.brave_news.models.FeedItemCard;
+import org.chromium.chrome.browser.brave_news.models.FeedItemsCard;
+
+public class BraveNewsUtils {
```

Comment:
This class doesn't have any non-static field, so it is possible remove ctor `public BraveNewsUtils() {}` ,  make `public String getPromotionIdItem` and update places where it is used

Reply:
done and replaced https://github.com/brave/brave-core/blob/45ef2a3938e4aba8485b6e92d1e5c523fc021552/android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java#L1181

Reply:
thanks :+1: